src/app/productConfig/productConfigForm/productConfigForm.tpl.html

```diff
@@ -1,4 +1,11 @@
 <form novalidate name="$ctrl.itemConfigForm" ng-submit="$ctrl.saveGiftToCart()">
+  <div class="row">
+    <span class="u-floatRight" style="margin-top: -16px">
+      {{'SECURE'}}
```

Comment:
I think you need the `translate` filter.

```suggestion
      {{'SECURE' | translate}}
```